docs/databases/introduction.md

```diff
@@ -1,7 +1,34 @@
 ---
-id: databases
-title: Setting up our database
-sidebar_label: Databases 
+id: overview
+title: Overview
+sidebar_label: Overiew 
 ---
 
-## TODO
\ No newline at end of file
+At the core of a CRUD API is the database that Graphback interacts with by providing a rich set of tools and APIs that help you get your database working with your GraphQL application in minutes.
+
+Graphback has a [CRUD abstraction](../crud/overview) with a number of out-of-the-box implementations for [PostgreSQL](postgres.md), [MongoDB](mongodb.md) and [SQLite](sqlite.md) databases.
+
+Additionally, Graphback integrates with [`graphql-migrations`](../graphql-migrations/intro.md) so that you can migrate your databases using your business models defined in the GraphQL schema. 
```

Comment:
```suggestion
For PostgreSQL, Graphback integrates with [`graphql-migrations`](../graphql-migrations/intro.md) so that you can migrate your databases using your business models defined in the GraphQL schema. 
```

Reply:
Ah.. Ignore me